src/components/player/PlayerHeroWinRate.vue

```diff
@@ -184,7 +219,7 @@ export default defineComponent({
         };
         resp.push(playerWinRate);
       }) || [];
-      return resp;
+      return resp.sort((a, b) => b.numbers_by_race[ERaceEnum.TOTAL].total - a.numbers_by_race[ERaceEnum.TOTAL].total);
```

Comment:
How will this interact with paginated data?

Reply:
We always have all the data up-front. The pagination is frontend.